backend/src/graphql/resolvers/mutations.js

```diff
@@ -0,0 +1,24 @@
+const bcrypt = require('bcrypt');
+const User = require('../../models/user');
+
+module.exports = {
+  register: async (_, { user }) => {
+    // TODO Valid email format
```

Comment:
Are you doing validation all at once in a separate PR?
Might be worth doing straight away